src/theme/components/index.tsx

```diff
@@ -467,17 +467,19 @@ export const SpinnerSVG = styled.svg`
   ${SpinnerCss}
 `
 
-const BackArrowLink = styled(StyledInternalLink)`
-  color: ${({ theme }) => theme.textPrimary};
-`
-export function BackArrow({ to }: { to: string }) {
+export function BackArrowLink({ to }: { to: string }) {
   return (
-    <BackArrowLink to={to}>
-      <ArrowLeft />
-    </BackArrowLink>
+    <StyledInternalLink to={to}>
+      <BackArrowIcon />
+    </StyledInternalLink>
   )
 }
 
+export const BackArrowIcon = styled(ArrowLeft)`
```

Comment:
Where is `BackArrowIcon` used? I can't find a reference in the diff.

Reply:
it was used before the design changed, but this refactor is the suggestion you made in your first review

Reply:
Makes sense, but my question is why do we need to `export` it outside of this module?

Reply:
https://github.com/Uniswap/interface/blob/main/src/nft/components/collection/TransactionCompleteModal.tsx#L378

Reply:
oh wait, that's a different icon! i think i can remove the export